python/lsst/ip/isr/isrTaskLSST.py

```diff
@@ -1061,32 +1061,21 @@ def getBrighterFatterKernel(self, detector, bfKernel):
             else:
                 raise RuntimeError("Failed to extract kernel from new-style BF kernel.")
         elif bfKernel.level == 'AMP':
-            self.log.warning("Making DETECTOR level kernel from AMP based brighter "
-                             "fatter kernels.")
+            self.log.info("Making DETECTOR level kernel from AMP based brighter "
+                          "fatter kernels.")
```

Comment:
Thank you for making this not a warning.  Is this even an `info` or should it be a `debug`?

Reply:
This honestly should become a default, but might be out of scope of the mock BF ticket

Reply:
Should we just get rid of the brighter fatter kernel `bfk.level` parameter entirely in this ticket? This would extend this ticket beyond ip_isr.

Reply:
I wouldn't change the bfk format, that does seem like a bridge too far.  I'm just saying that because this is such a standard operation I'm not sure if it needs to be logged at all at info level.  Logging below that BF is being applied is important, but this just seems ... extraneous.

Reply:
For clarity, we I think we should keep it in, until we make this the only option.